app/presenters/brexit_landing_page_presenter.rb

```diff
@@ -50,22 +50,8 @@ def translation_links
 
 private
 
-  SWITCHOVER_TIME = Time.zone.parse("2020-01-31 23:00:00").in_time_zone
-
-  def time_based_intl
-    if before_switchover?
-      "brexit_landing_page"
-    else
-      "transition_landing_page"
-    end
-  end
-
-  def before_switchover?
-    Time.zone.now < SWITCHOVER_TIME
-  end
-
   def fetch_buckets
-    buckets = I18n.t("#{time_based_intl}.campaign_buckets")
+    buckets = I18n.t("transition_landing_page.campaign_buckets")
```

Comment:
somehow not in the scope of this PR, but should we have `t` instead of `I18n.t` for consistency?

Reply:
or is `t` available only as a view helper?

Reply:
Yeah, i checked this - so if you remove it here in the presenter it errors!!
However there appears to be some kind of override in helpers that must do something like:
`t = I18n.t`, which is a bit annyoing.

Would kind of like to throw it out to the community and ask what a good standard should be then write a rubocop rule for it?

`t()` is concise, but seems a bit orphan `I18n.t()` always seems clear what it is and where it comes from... so i might stay towards the more verbose?

Reply:
Ok, so it looks like `t` is an ActionView helper, so we can't use `t` outside of views.

https://apidock.com/rails/v5.2.3/ActionView/Helpers/TranslationHelper/translate